wfe/wfe.go

```diff
@@ -1115,6 +1121,15 @@ func (wfe *WebFrontEndImpl) Registration(ctx context.Context, logEvent *requestE
 		return
 	}
 
+	if wfe.AllowAccountDeactivation && update.Status != "" {
```

Comment:
is there a reason to explicitly check for `update.Status != ""` here to silently return instead of not checking it and allowing the inner `if update.Status != core.StatusDeactivated` to return a `probs.Malformed` for `update.Status == ""`? Seems like the second might be clearer to someone that submitted an empty status by mistake vs wondering why the update didn't take but no error was produced.